internal/distro/rhel8/pipelines.go

```diff
@@ -670,6 +670,16 @@ func osPipeline(t *imageType,
 	}
 
 	if t.rpmOstree {
+		p.AddStage(osbuild.NewSystemdJournaldStage(
+			&osbuild.SystemdJournaldStageOptions{
+				Filename: "10-persistent.conf",
+				Config: osbuild.SystemdJournaldConfigDropin{
+					Journal: osbuild.SystemdJournaldConfigJournalSection{
+						Storage: osbuild.StoragePresistent,
+					},
+				},
+			},
+		))
```

Comment:
For correctness sake, it should be before the SELinux stage, but `NewOSTreePrepTreeStage` needs to be the last one.